test/functional/specs/C14412.js

```diff
@@ -1,41 +0,0 @@
-import fixtureFactory from "../helpers/fixtureFactory";
-import configureAlloyInstance from "../helpers/configureAlloyInstance";
-import {
-  compose,
-  orgMainConfigMain,
-  consentPending
-} from "../helpers/constants/configParts";
-
-const config = compose(
-  orgMainConfigMain,
-  consentPending
-);
-
-fixtureFactory({
-  title:
-    "C14412: While user is changing consent preferences, other requests should be queued"
-});
-
-test.meta({
-  ID: "C14412",
-  SEVERITY: "P0",
-  TEST_RUN: "Regression"
-});
-
-test("Test C14412: While user is changing consent preferences, other requests should be queued", async t => {
-  await configureAlloyInstance("alloy", config);
-  await t.eval(() => window.alloy("setConsent", { general: "in" }));
-  await t.eval(() => {
-    window.alloy("setConsent", { general: "out" });
-    // don't wait for setConsent to be done, but wait for the command to be run.
-    return undefined;
-  });
-  const errorMessage = await t.eval(() =>
-    window
-      .alloy("sendEvent", { data: { a: 1 } })
-      .then(() => undefined, e => e.message)
-  );
-
-  await t.expect(errorMessage).ok("Expected the event command to be rejected");
-  await t.expect(errorMessage).contains("user declined consent");
-});
```

Comment:
I deleted this test because it was a duplicate of what we're doing in C14414.js. I'll delete the corresponding test case in test rail assuming this PR gets merged like this.